code/modules/mob/living/simple_animal/hostile/mining_mobs/hivelord.dm

```diff
@@ -301,7 +301,7 @@
 	H.dna.add_mutation(DWARFISM)
 
 /obj/effect/mob_spawn/human/corpse/damaged/legioninfested/Initialize(mapload)
-	var/type = pick_weight(list("Miner" = 66, "Ashwalker" = 10, "Golem" = 10,"Clown" = 10, pick(list("Shadow", "YeOlde","Operative", "Cultist")) = 4))
+	var/type = pick_weight(list("Miner" = 66, "Ashwalker" = 10, "Golem" = 10, "Clown" = 10, pick(list("Shadow", "YeOlde","Operative", "Cultist")) = 4))
```

Comment:
giving a merge conflict to armhulen smh

Reply:
I painfully went through every single mention of every single job.